lib/application.dart

```diff
@@ -11,17 +12,18 @@ class Application extends StatelessWidget {
     return BlocProvider<ApplicationBloc>(
       bloc: ApplicationBloc(),
       child: MaterialApp(
+        debugShowCheckedModeBanner: false,
```

Comment:
What is this property doing?

Reply:
this will remove the debug banner in debug mode